### PR TITLE
chore: add Artur to the list of package authors

### PR DESCRIPTION
**Pull request by kbsali**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,6 +13,11 @@
             "name": "Kevin Saliou",
             "email": "kevin@saliou.name",
             "homepage": "http://kevin.saliou.name"
+        },
+        {
+            "name": "Artur Weigandt",
+            "email": "artur@wlabs.de",
+            "homepage": "http://kevin.saliou.name"
```
Comment (Art4) [inline composer.json]:
```suggestion
            "homepage": "https://wlabs.de"
```

**Review by Art4 — CHANGES_REQUESTED** [review]
Thank you. 😊

File: composer.json (hunk 1 above)
Comment (kbsali) [reply composer.json]: oops :sweat_smile:

**Review by Art4 — APPROVED** [review]

**Comment by kbsali** [comment]
@Art4 I have tried to setup codecov but it fails at generating the coverage file... not sure how to fix this :pray:

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -38,11 +37,11 @@ jobs:
         uses: "ramsey/composer-install@v2"
 
       - name: Run tests
-        run: vendor/bin/phpunit --coverage-text
+        run: vendor/bin/phpunit --coverage-clover coverage.xml .
```
Comment (Art4) [inline .github/workflows/tests.yml]:
```suggestion
        run: vendor/bin/phpunit --coverage-clover .phpunit.cache/clover.xml
```

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -38,11 +37,11 @@ jobs:
...
 
       - name: Upload coverage reports to Codecov
         uses: codecov/codecov-action@v3
         env:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```
Comment (Art4) [inline .github/workflows/tests.yml]:
```suggestion
      - name: Upload coverage reports to Codecov
        if: ${{ success() && matrix.matrix.php-versions == '8.2' }}
        uses: codecov/codecov-action@v3
        with:
          files: ./.phpunit.cache/clover.xml
          fail_ci_if_error: true
          verbose: true
```

**Comment by Art4** [comment]
> @Art4 I have tried to setup codecov but it fails at generating the coverage file... not sure how to fix this 🙏

I've made two suggestions to setup Codecov properly.

**Review by Art4 — CHANGES_REQUESTED** [review]
I've made two suggestions to setup Codecov properly.

**Comment by kbsali** [comment]
:+1:

**Pull request merged**